hazelcast/src/test/java/com/hazelcast/map/ExpirationTimeTest.java

```diff
@@ -344,8 +344,7 @@ public void testExpirationTime_whenMaxIdleTime_isSmallerThan_TTL() {
         map.put(1, 1, 100, TimeUnit.SECONDS);
 
         EntryView<Integer, Integer> entryView = map.getEntryView(1);
-        long lastAccessTime = entryView.getLastAccessTime();
-        long delayToExpiration = lastAccessTime + TimeUnit.SECONDS.toMillis(10);
+        long delayToExpiration = TimeUnit.SECONDS.toMillis(10);
 
         // lastAccessTime is zero after put, we can find expiration by this calculation
```

Comment:
This comment is stale. Could you update it?

Reply:
removed.